recipes/kafka-connect.rb

```diff
@@ -1,9 +1,10 @@
 # frozen_string_literal: true
+
 include_recipe 'confluent::_install'
 connect_class = node['confluent']['kafka-connect']['distributed_mode'] ? node['confluent']['kafka-connect']['distributed_class'] : node['confluent']['kafka-connect']['standalone_class']
 
 confluent_extracted_dir = File.join(node['confluent']['install_dir'], "confluent-#{node['confluent']['version']}")
-connect_jdbc_jar_dir = File.join confluent_extracted_dir, 'share/java/kafka-connect-all'
+connect_share_dir = File.join confluent_extracted_dir, 'share', 'java/kafka-connect-all'
```

Comment:
somehow didn't notice that, https://github.com/bbaugher/confluent/pull/24/commits/d99877ac6e073ef962171f4767388c8aa081720f